src/services/index.js

```diff
@@ -1,75 +1,79 @@
 const mongoose = require('mongoose');
-const fileStorage = require('./fileStorage');
-const link = require('./link');
-const news = require('./news');
-const content = require('./content');
-const calendar = require('./calendar');
-const ltiTool = require('./ltiTool');
-const school = require('./school');
-const system = require('./system');
-const lesson = require('./lesson');
-const analytics = require('./analytics');
+
```

Comment:
In dieser Datei habe ich nur alle imports mal nach Name sortiert und linter issues behoben